tests/dash/test_dash_acl.py

```diff
@@ -87,10 +87,12 @@ def __setitem__(self, key: str, pairs: Union[dict, list, tuple]):
             pairs_str.append((to_string(k), to_string(v)))
         self.table.set(key, pairs_str)
         self.keys.add(key)
+        time.sleep(3)
```

Comment:
>sleep [](http://example.com/codeflow?start=13&length=5)

Since it is in the lowest level, it may dramatically slow down the testrun, assuming there will lots of set operations.

Reply:
Improve by change sleep 1second.
I checked the code, there will be 5-10 set/delete operation for every test case.